app/services/onlinetesting/OnlineTestExpiryService.scala

```diff
@@ -42,16 +43,13 @@ class OnlineTestExpiryServiceImpl(
   newHeaderCarrier: => HeaderCarrier
 )(implicit executor: ExecutionContext) extends OnlineTestExpiryService {
 
-  private final val ExpiredStatus = "ONLINE_TEST_EXPIRED"
   private implicit def headerCarrier = newHeaderCarrier
 
   override def processNextExpiredTest(): Future[Unit] = {
-    // TODO FAST STREAM FIX ME
-    Future.successful(Unit)
-    //otRepository.nextApplicationPendingExpiry.flatMap {
-    //  case Some(expiringTest) => processExpiredTest(expiringTest)
-    //  case None => Future.successful(())
-    //}
+    otRepository.nextApplicationPendingExpiry.flatMap {
+      case Some(expiringTest) => processExpiredTest(expiringTest)
```

Comment:
if we get an expiringTest, as I wrote before, it is weird that the method is called nextApplication....